node/forcedroid.js

```diff
@@ -392,17 +393,35 @@ function getTemplateSourceFilePaths(config) {
 
 function copyFromSDK(packageSdkRootDir, targetDir, srcDirRelative) {
     console.log('Copying ' + srcDirRelative + '.');
-    var destDir = path.join(targetDir, path.basename(packageSdkRootDir), path.dirname(srcDirRelative));
     var srcDir = path.join(packageSdkRootDir, srcDirRelative);
-    if (!fs.existsSync(path.join(destDir, path.basename(srcDirRelative)))) {
-        shelljs.mkdir('-p', destDir);
-        shelljs.cp('-R', srcDir, destDir);
-        if (shelljs.error()) {
-            console.log('There was an error copying ' + srcDirRelative + ' from \'' + packageSdkRootDir + '\' to \'' + destDir + '\': ' + shelljs.error());
-            process.exit(5);
+    var destDir;
+    if (fs.lstatSync(srcDir).isDirectory()) {
+        destDir = path.join(targetDir, path.basename(packageSdkRootDir), srcDirRelative);
+        if (!fse.existsSync(path.join(destDir, path.basename(srcDirRelative)))) {
+            try {
+            fse.mkdirsSync(destDir);
+            fse.copySync(srcDir, destDir);
+            } catch (err) {
+                console.error(err);
+            }
+        }
+        else {
+            console.log(outputColors.cyan + 'INFO:' + outputColors.reset + ' ' + srcDirRelative + ' already exists.  Skipping copy.');
+        }
+    }
+    else {
+        destDir = path.join(targetDir, path.basename(packageSdkRootDir), path.dirname(srcDirRelative));
+        if (!fse.existsSync(path.join(destDir, path.basename(srcDirRelative)))) {
+            shelljs.mkdir('-p', destDir);
```

Comment:
shelljs of 0.7.0 version has issue for folder copy, while fse copy file hit permission for copying file, that's why we cannot use shelljs or fse for both file and folder copy